### PR TITLE
Mark instrumented sources with precaution message

### PR DESCRIPTION
**Pull request by blitz-1306**
## Tasks
A part of changes to address #119.

## Changes
- [x] Introduced precaution message to ask user to not edit instrumented files directly. Message will be preprended to generated sources.
- [x] Updated source location computation logic to respect precaution message (shift AST node locations).
- [x] Updated affected test snapshots and tweaked few test suites.

## Notes
A lot of test snapshots are updated by this PR.

Regards.

### CODE REVIEW

File: src/bin/scribble.ts [hunk 1]
```diff
@@ -629,11 +629,16 @@ if ("version" in options) {
         }
     }
 
+    const instrumentationMarker =
+        "/// This source is auto-generated by Scribble and shouldn't be edited directly.\n" +
```
Comment (cd1m0) [inline src/bin/scribble.ts:633]: nit: source -> file would be a little cleaner

**Review by cd1m0 — APPROVED** [review]
LGTM. Just one nit - not a blocker

File: src/bin/scribble.ts (hunk 1 above)
Comment (blitz-1306) [reply src/bin/scribble.ts:633]: Will fix

Comment (blitz-1306) [reply src/bin/scribble.ts]: Addressed in https://github.com/ConsenSys/scribble/pull/127/commits/b76dea9190f54c5098a65840961143fdb78b17d8

**Review by cd1m0 — APPROVED** [review]
lgtm

**Pull request merged**